swap/types.go

```diff
@@ -34,7 +31,6 @@ type ChequeParams struct {
 }
 
 // Cheque encapsulates the parameters and the signature
-// TODO: There should be a request cheque struct that only gives the Serial
```

Comment:
Why is this one removed? Is it already done? Is there an issue?

Reply:
As @mortelli mentioned: ```swap/types.go:34: remove, we no longer request cheques.```
No new issue was created as this is (also in my opinion) an outdated TODO. If you think otherwise, please tell!

Reply:
We don't request cheques any longer.